packages/tables/resources/views/components/button.blade.php

```diff
@@ -101,8 +101,8 @@
 @elseif ($tag === 'a')
     <a
         @if ($tooltip)
-            x-data="{}"
-            x-tooltip.raw="{{ $tooltip }}"
+            x-data="tooltip(@js($tooltip))"
+            x-tooltip="tooltip"
```

Comment:
can we just typeof tooltip !== 'undefined'  check
as the concern is on the new `Alpine.data('tooltip')`
and keep `x-tooltip.raw` as fallback for those not using admin panel

`
x-data="typeof tooltip !== 'undefined' ? tooltip(@js($tooltip)) : {}"
x-tooltip.raw="{{ $tooltip }}"
x-tooltip="typeof tooltip !== 'undefined' ? tooltip : null"
`

however i also don't have standalone table project, but i simulated it seems fine

Reply:
So multiple `x-tooltip` attributes on the same element works?